resources/todomvc/vanilla-examples/javascript-es5/.eslintrc

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Could we make this inherit eslint and prettier settings from the project root by removing it from the relevant ignore files? That would make each individual patch smaller and central changes easier.

Reply:
ah I see what you mean. 
I like that each workload is independent at the moment. 

The potential disadvantage of having S3 format all workloads is that it might surface issues on workloads that I am not currently working on.

Reply:
I would opt in folder by folder - see for example https://github.com/WebKit/Speedometer/pull/81/files#diff-b640b344ee7f3f03d2a443795a5d0708ef50e2e6e34214109ab2aad13ad6ba98L3

Reply:
done 👍